files/image_config/logrotate/logrotate-rsyslog.j2

```diff
@@ -84,10 +91,20 @@
         done
     endscript
     postrotate
+{% if namespaces > 1 %}
+{% for ns in range(namespaces) %} 
+        if [ $(echo $1 | grep -c "/var/log/swss{{ns}}/") -gt 0 ]; then
+            pgrep -x orchagent | xargs /bin/kill -HUP 2>/dev/null || true
```

Comment:
will this kill all orchagent instances?

Reply:
you probably do not need to do template here since, logrotate will pass the absolute path to the log file to this postrotate script, and from that absoluate path, you will know whethere it is multiasic or not, and which asic id it is if it is multi-asic. 

moreover, the orchagent should

Reply:
maybe you should get into the container and do the pkill